### PR TITLE
Return error from exec

### PR DESCRIPTION
**Pull request by smarterclayton**
`oc ex dockerbuild` was not recording failures from exec.

@mfojtik

[test]

### CODE REVIEW

File: pkg/util/docker/dockerfile/builder/client.go [hunk 1]
```diff
@@ -312,11 +312,20 @@ func (e *ClientExecutor) Run(run Run, config docker.Config) error {
...
-	return err
+	}); err != nil {
+		return err
+	}
+	status, err := e.Client.InspectExec(exec.ID)
+	if err != nil {
+		return err
+	}
+	if status.ExitCode != 0 {
+		return fmt.Errorf("running '%s' failed with exit code %d", strings.Join(args, " "), status.ExitCode)
```
Comment (mfojtik) [inline pkg/util/docker/dockerfile/builder/client.go]: %q ?

**Comment by mfojtik** [comment]
one nit, lgtm :-)

File: pkg/util/docker/dockerfile/builder/client.go (hunk 1 above)
Comment (smarterclayton) [inline pkg/util/docker/dockerfile/builder/client.go]: Args will have double quotes, so didn't want to put doubles inside of doubles.

**Comment by smarterclayton** [comment]
This now highlights a break in openshift/openvswtich, where the COPY doesn't appear to be correct.

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
Please review second commit, going to merge because openvswitch is broken.  [merge]

**Pull request merged**